src/pages/boston/speakers.jsx

```diff
@@ -43,11 +43,8 @@ const SpeakersPage = () => {
             <div className="col-full lg:col-6">
               <h1 className="typo-h2 mb-4">More than 25 speakers to look forward to</h1>
               <p className="typo-body max-w-xl mb-6">
-                The call for speakers and posters has been extended until August 11.
+                The call for abstracts is now closed. The final abstracts will be accepted and presenters notified in mid August..
```

Comment:
```suggestion
                The call for abstracts is now closed. The final abstracts will be accepted and presenters notified in mid August.
```

Reply:
Maybe we can have some better text here - like "this page will be updated..." etc etc, rather than just saying abstracts have closed (the reason for which is less obvious here).